src/core/public/chrome/ui/header/collapsible_nav.tsx

```diff
@@ -197,7 +196,7 @@ export function CollapsibleNav({
       outsideClickCloses={false}
     >
       <EuiFlexItem className="eui-yScroll">
-        <CollapsibleNavHeader workspaces={workspaces} />
+        {collapsibleNavHeaderRender ? collapsibleNavHeaderRender() : <CollapsibleNavHeader />}
```

Comment:
By your PR, `CollapsibleNavHeader` would simply be a react component, I would suggest moving `CollapsibleNavHeader` to `collapsible_nav.tsx`

Reply:
Good point, I will update